enigma/src/main/java/org/quiltmc/enigma/api/translation/representation/entry/MethodEntry.java

```diff
@@ -49,29 +50,45 @@ public boolean isConstructor() {
 	}
 
 	/**
-	 * Creates an iterator of all parameters in this method. Unmapped args will have no name, and javadoc is ignored.
+	 * Get a list of all parameters in this method.
+	 *
 	 * @param index the entry index
-	 * @param deobfuscator a translator
-	 * @return an iterator of this method's parameters
+	 * @return a list of this method's parameters
 	 */
-	public Iterator<LocalVariableEntry> getParameterIterator(EntryIndex index, Translator deobfuscator) {
+	public List<LocalVariableEntry> getParameters(EntryIndex index) {
 		List<LocalVariableEntry> parameters = new ArrayList<>();
-
-		MethodDescriptor desc = this.getDesc();
 		AccessFlags flags = index.getMethodAccess(this);
 
-		if (desc != null && flags != null) {
-			int argIndex = flags.isStatic() ? 0 : 1;
+		if (flags != null) {
+			int i = flags.isStatic() ? 0 : 1;
 
-			for (ArgumentDescriptor arg : desc.getArgumentDescs()) {
-				LocalVariableEntry argEntry = new LocalVariableEntry(this, argIndex, "", true, null);
-				LocalVariableEntry translatedArgEntry = deobfuscator.translate(argEntry);
+			for (ArgumentDescriptor arg : this.descriptor.getArgumentDescs()) {
+				LocalVariableEntry argEntry = new LocalVariableEntry(this, i, "", true, null);
+				parameters.add(argEntry);
 
-				parameters.add(translatedArgEntry == null ? argEntry : translatedArgEntry);
-				argIndex += arg.getSize();
+				i += arg.getSize();
 			}
 		}
 
+		return parameters;
+	}
+
+	/**
+	 * Creates an iterator of all parameters in this method, also doing translation. Unmapped args will have no name, and javadoc is ignored.
```

Comment:
```suggestion
	 * Creates an iterator of all parameters in this method, also doing translation. Unmapped args will have an empty name, and javadoc is ignored.
```
now that I've changed the other one I think this is better because it tells you it's an empty string instead of leaving you wondering if it's null or empty